etc/lfcd.fish

```diff
@@ -11,17 +11,8 @@
 #
 # You may put this in a function called fish_user_key_bindings.
 
-function lfcd
-    set tmp (mktemp)
+function lfcd --wraps="lf" --description="lf - Terminal file manager (changing directory on exit)"
     # `command` is needed in case `lfcd` is aliased to `lf`
-    command lf -last-dir-path=$tmp $argv
-    if test -f "$tmp"
-        set dir (cat $tmp)
-        rm -f $tmp
-        if test -d "$dir"
-            if test "$dir" != (pwd)
-                cd $dir
-            end
-        end
-    end
+    command lf -print-last-dir "$argv" \
```

Comment:
`$argv` shouldn't be quoted, as this will join all the arguments into a single string. This means something like `lfcd -log ~/lf.log` will fail as it treats `-log ~/lf.log` as a single argument.

Reply:
Thank you, fixed it